bin/scheduler.ml

```diff
@@ -121,6 +122,26 @@ let main capnp secrets_dir pools prometheus_config state_dir default_clients =
 
 (* Command-line parsing *)
 
+let main ((capnp, secrets_dir, pools, prometheus_config, state_dir, default_clients, install, remove), arguments) =
+  let (name, display, text) = ("ocluster-scheduler", "OCluster Scheduler", "Manage build workers") in
+  match install, remove with
+  | (true, true) -> `Error (true, "That's clever!")
```

Comment:
Please avoid unhelpful error messages. Assume this message has resulted from some complex script, or a bug introduced long after this code was written, and that the problem is not obvious to the person seeing the error.

Reply:
Actually, I think that may be an accidentally-copied artefact from my sample code on the cmdliner change...!

Reply:
Ouh, I thought it was typical british error reporting... Changed to something sane.

Reply:
It was typical British error reporting 😉